services/dauth-service/src/core/auth_vectors.rs

```diff
@@ -272,9 +272,8 @@ pub async fn backup_auth_vector_used(
     )
     .await?;
 
-    let (_, mut backup_networks) =
+    let (_, backup_networks) =
         clients::directory::lookup_user(context.clone(), &user_id).await?;
-    backup_networks.retain(|id| id != backup_network_id);
```

Comment:
Just to check my understanding, the issue here is that we were not sending the backup network that used the auth a key share itself?

Reply:
Yes, exactly. That was my misunderstanding from before, I forgot to change it here.